.claude/scripts/poll-pr.sh

```
@@ -1,191 +1,523 @@
+    echo "--- PR Details ---"
+    echo "$pr_meta" | jq -r 'to_entries[] | "  \(.key): \(.value)"' 2>/dev/null || echo "$pr_meta"
+    echo ""
+
+    echo "--- CI Check Runs ---"
+    gh pr checks "$PR_NUMBER" --repo "$REPO" 2>/dev/null || echo "  (no checks found)"
+    echo ""
+
+    echo "--- CodeRabbit Review ---"
+    if [ "$cr_state" != "NONE" ] && [ -n "$cr_state" ]; then
+      printf "  State: %s  (at: %s)\n" "$cr_state" "$cr_at"
+      if ! $cr_is_current; then
```

Shellcheck (0.11.0)

[warning] cr_is_current is referenced but not assigned.

(SC2154)